code/modules/client/preferences.dm

```diff
@@ -1583,6 +1585,38 @@ Slots: [job.spawn_positions] [job.round_contrib_points ? "RCP: +[job.round_contr
 							return
 						voice_pitch = new_voice_pitch
 
+				if("voice_type_mumble")
+					var/static/list/selectable_voices = list(
+						"neutral_01",
+						"male_01",
+						"male_02",
+						"male_03",
+						"male_04",
+						"male_05",
+						"male_06",
+						"male_07",
+						"female_01",
+						"female_02",
+						"female_03",
+						"female_04",
+						"female_05",
+						"female_06",
+						"female_07"
+					)
+					var/list/voice_choices = list("None")
+					for(var/voice in selectable_voices)
+						if(voice in pref_species.voice_type_mumble)
```

Comment:
`GLOB.all_voice_type_list` instead of `pref_species.voice_type_mumble)` to access the global list you made.